templates/page.values.php

```diff
@@ -41,11 +41,14 @@
 			</li>
 			<li>
 				<span class="icon-shared"></span>
-				<h3><?php p($l->t('Open Standards and Interoperability')); ?></h3>
+				<h3><?php p($l->t('Open standards and interoperability')); ?></h3>
 			</li>
 			<li>
 				<span class="icon-user"></span>
-				<h3><?php p($l->t('100%% Open Source & community-focused')); ?></h3>
+				<h3><?php 
+					// TRANSLATORS NOTE: The characters %% should be left unchanged. This is a workaround for PHP code.
+					p($l->t('100%% open source & community-focused'));
```

Comment:
This is not a "workaround", just the way translations works :)
We could also just use plain-text like :shrug: 

```suggestion
					p($l->t('One-hundred percent open source & community-focused'));
```

Reply:
I shortened the comment. For translators, this information is all they need to be wary of the `%%` characters.